### PR TITLE
Touch-ups to GC config knobs

### PR DESCRIPTION
**Pull request by gewarren**
- touch ups to core config knobs (feedback from PR #15844)
- also adds new page for GCLOHThreshold option for app.config

Preview link: https://review.docs.microsoft.com/en-us/dotnet/core/run-time-config/garbage-collector?branch=pr-en-us-15914

### CODE REVIEW

**Comment by Maoni0** [comment]
LGTM! BTW, I apologize - I forgot to mention that for the GCLOHThreshold config we should note that you can only specify a value that's larger than the default (which is 85000 bytes).

**Review by mairaw — APPROVED** [review]
LGTM modulo @Maoni0's comment

File: docs/core/run-time-config/garbage-collector.md [hunk 1]
```diff
@@ -188,8 +198,12 @@ For more information about some of these settings, see the [Middle ground betwee
 
 | | Setting name | Values | Version introduced |
 | - | - | - | - |
-| **runtimeconfig.json** | `System.GC.LOHThreshold` | *size in bytes* | .NET Core 1.0 |
-| **Environment variable** | `COMPlus_GCLOHThreshold` | *size in bytes* | .NET Core 1.0 |
+| **runtimeconfig.json** | `System.GC.LOHThreshold` | *decimal value* | .NET Core 1.0 |
+| **Environment variable** | `COMPlus_GCLOHThreshold` | *hexadecimal value* | .NET Core 1.0 |
+| **app.config for .NET Framework** | [GCLOHThreshold](../../framework/configure-apps/file-schema/runtime/gclohthreshold-element.md) | *size in bytes* | .NET Framework 4.8 |
```
Comment (pkulikov) [inline docs/core/run-time-config/garbage-collector.md]: Size in bytes: is it decimal?

**Comment by gewarren** [comment]
Thanks all for the reviews.

**Pull request merged**